<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMMyApp</class>
 <widget class="QWidget" name="SMMyApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="AppTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>840</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="tools" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QToolButton" name="tbAnzhuang">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>4</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbXiezai">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>4</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbSdcard">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>4</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbDaochu">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>4</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbShuaxin">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>4</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AppTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">apptablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
